toolchains/cc/cc.bzl

```diff
@@ -383,7 +390,14 @@ def nixpkgs_cc_configure(
         nix_expr = nix_file_content
 
     if attribute_path and nix_expr == None:
-        fail("'attribute_path' requires one of 'nix_file' or 'nix_file_content'", "attribute_path")
+        nixopts.extend([
+            "--argstr",
+            "ccType",
+            "ccTypeExpression",
+            "--arg",
```

Comment:
Perhaps this would be a bit simpler?
```
@@ -388,17 +388,11 @@ def nixpkgs_cc_configure(
         nix_file_deps.append(nix_file)
     elif nix_file_content:
         nix_expr = nix_file_content
-
-    if attribute_path and nix_expr == None:
-        nixopts.extend([
-            "--argstr",
-            "ccType",
-            "ccTypeExpression",
-            "--arg",
-            "ccExpr",
-            "(import <nixpkgs> {{}}).{0}".format(attribute_path),
-        ])
     elif attribute_path:
+        nix_expr = "(import <nixpkgs> {{}}).{0}".format(attribute_path)
+        attribute_path = None
+
+    if attribute_path:
         nixopts.extend([
             "--argstr",
             "ccType",


```
I think it's equivalent but the logic is a little convoluted so I may be wrong.

Reply:
Not sure how to make a GitHub suggested change that I've seen people do sometimes. Here's an actual commit though:
https://github.com/tweag/rules_nixpkgs/commit/841908b4288481a0a022193d76bb251170212d1e